.build/ory/kratos/email-password/kratos.yml

```diff
@@ -4,70 +4,53 @@ dsn: memory
 
 serve:
   public:
-    base_url: http://localhost:3000/
+    base_url: http://localhost:3000/identity/ory/kratos/public
     cors:
       enabled: true
   admin:
-    base_url: http://localhost:4434/
+    base_url: http://kratos:4434/
 
 selfservice:
   default_browser_return_url: http://localhost:3000/
   whitelisted_return_urls:
     - http://localhost:3000
 
   methods:
-    link:
-      enabled: true
-
```

Comment:
Used for sending recovery and verification emails.

Reply:
Addressed in ff3914cf670151f476fbe9bdf9b747d578aec552